doc/specs/stdlib_strings.md

```diff
@@ -192,3 +192,64 @@ program demo
   print'(a)', ends_with("pattern", "pat")  ! F
 end program demo
 ```
+
+
+<!-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -->
+### `slice`
+
+#### Description
+
+Extracts the characters from the defined region of the input string.  
+Argument `first` and `last` defines the region for the function `slice` to operate.  
```

Comment:
We should also state that in case of invalid inputs, i.e. `first < 1` or `last > len(str)`, the function assumes the default limits.